dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/MetadataSchemaRestRepository.java

```diff
@@ -93,6 +93,8 @@ protected MetadataSchemaRest createAndReturn(Context context)
         // validate fields
         if (isBlank(metadataSchemaRest.getPrefix())) {
             throw new UnprocessableEntityException("metadata schema name cannot be blank");
+        } else if (metadataSchemaRest.getPrefix().contains(".")) {
```

Comment:
We also should check for commas, spaces and underscores... as those are all invalid in the UI.